src/db/models/draftarticles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Draftarticles = sequelize.define('Draftarticles', {
```

Comment:
what's the purpose of this table, yet an article's table was created from the 'create articles' task.
Please use the same table, don't create another one.

Reply:
I used before rebasing develop, let's delete it